src/containers/Application/NavigationBox.tsx

```diff
@@ -16,71 +19,65 @@ interface ModalProps {
   const [showModal, setShowModal] = useState<ModalProps>({
     open: false,
     message: '',
     title: '',
   })
-  const { query, push } = useRouter()
-  const { serialNumber, sectionCode, page } = query
 
-  const currentSection = templateSections.find(({ code }) => code === sectionCode)
+  const nextSection = templateSections.find(({ index }) => index === currentSection.index + 1)
+  const previousSection = templateSections.find(({ index }) => index === currentSection.index - 1)
 
-  const isFirstPage = checkFirstPage({
-    sectionCode: sectionCode as string,
-    currentPage: Number(page),
-    templateSections,
-  })
+  const isFirstPage = currentPage - 1 === 0 && !previousSection
+  const isLastPage = currentPage + 1 > currentSection.totalPages && !nextSection
 
-  const isLastPage = checkLastPage({
-    sectionCode: sectionCode as string,
-    currentPage: Number(page),
-    templateSections,
-  })
+  const { push } = useRouter()
+  const sendToPage = (section: string, page: number) =>
+    push(`/applications/${serialNumber}/${section}/Page${page}`)
 
-  const changePageProps: changePageProps = {
-    currentPage: Number(page),
-    currentSection: currentSection as TemplateSectionPayload,
-    templateSections,
-    sendToPage: (section: string, page: number) =>
-      push(`/applications/${serialNumber}/${section}/Page${page}`),
-    sendToSummary: () => push(`/applications/${serialNumber}/summary`),
-    validateCurrentPage,
+  const previousButtonHandler = (_: any) => {
+    const previousPage = currentPage - 1
+    if (previousPage === 0) {
+      // Will check if previous page is in a other section
+      previousSection
+        ? sendToPage(previousSection.code, previousSection.totalPages)
+        : console.log('Problem to load previous page (not found)!')
+    } else sendToPage(currentSection.code, previousPage)
   }
 
-  return currentSection ? (
-    <>
-      <Grid columns={3}>
-        <Grid.Row>
-          <Grid.Column>
-            {!isFirstPage && (
-              <PageButton
-                title="Previous"
-                type="left"
-                onClicked={() => previousButtonHandler(changePageProps)}
-              />
-            )}
-          </Grid.Column>
-          <Grid.Column />
-          {/* Empty cell */}
-          <Grid.Column>
-            <PageButton
-              title={isLastPage ? 'Summary' : 'Next'}
-              type="right"
-              onClicked={() => {
-                if (!nextPageButtonHandler(changePageProps)) {
-                  setShowModal({ open: true, ...VALIDATION_FAIL })
-                }
-              }}
-            />
-          </Grid.Column>
-        </Grid.Row>
-      </Grid>
+  const nextPageButtonHandler = (_: any): void => {
+    // Run the validation on the current page
+    const status = validateCurrentPage()
+    if (!status) {
+      setShowModal({ open: true, ...VALIDATION_FAIL })
```

Comment:
should we show the same modal when future page / section is pressed (but it can't be navigated) ? or maybe we can remove 'selectable' look on pages that can't be navigated

Reply:
I think we can ask what Debs think and create a separated issue to do that. But that's a good point! I'll create a note in the board as placeholder for an issue.

Reply:
[Note](https://github.com/orgs/openmsupply/projects/6#card-50955479)

Reply:
Sure, it's just seemed weird to me that pages look like they are selectable, but nothing happens when they are pressed.